### ISSUE TITLE
Code of training PPO end-to-end

### ISSUE THREAD

**Issue by liuqi8827** [tianhaowuhz/human-assisting-dex-grasp#6]
Thanks for your great work.

Can you release the `train PPO end-to-end` code and other baseline codes compared in the paper?

**Comment by tianhaowuhz** [comment]
for PPO baseline, you can override following args,
'''
reward_type = "define your own reward"
sub_obs_type = "joint+fingertipjoint+wrist+objpcl"
cfg_train['setting']['action_type'] = "direct"
cfg_train['setting']['sub_action_type'] = "none"
cfg_train['policy']['pretrain_pointnet'] = "whether pretrain pointnet"
cfg_train['policy']['pointnet_version'] = 'choose type you want'
'''
For baselines about IBS and ILAD, I recommend you refer to their open source code, since in order to fit my env, I modify the code a lot, and it can be hard to understanding.

**Comment by liuqi8827** [comment]
Thanks a lot!
